psi4/driver/p4util/python_helpers.py

```diff
@@ -123,6 +123,14 @@ def _core_wavefunction_build(mol, basis=None):
 
 core.Wavefunction.build = _core_wavefunction_build
 
+def _core_wavefunction_get_scratch_filename(self, filenumber):
+    """ Given a wavefunction and a scratch file number, canonicalizes the name
+        so that files can be consistently written and read """
+    fname = os.path.split(os.path.abspath(core.get_writer_file_prefix(self.molecule().name())))[1]
+    psi_scratch = core.IOManager.shared_object().get_default_path()
+    return os.path.join(psi_scratch, fname + str(filenumber) + '.npy')
```

Comment:
Should we standardize the ending here, might be other filetypes written? IIRC the wavefunction readers/writers will find the right file ending.

Reply:
I agree.  I had to modify the wfn from_file to do the right thing, but I just pushed a change omiting the `npy` suffix.